packages/playground/src/dashboard/farms_view.vue

```diff
@@ -29,13 +29,11 @@ export default {
     const userFarms = ref();
     function handleFarmCreated() {
       farmsReload.value = !farmsReload.value;
-    }
-    watch(
-      () => farmsReload.value,
-      () => {
+      if (userFarms.value) {
         userFarms.value.reloadFarms = farmsReload.value;
```

Comment:
what is the idea of having two values got updated on reload, if that is not part of your issue, I will open one to revisit this logic

Reply:
this logic is removed in this pr, so comment is irrelevant